### PR TITLE
docs: add R client ref and examples

### PR DESCRIPTION
**Pull request by alespour**
This PR adds R client reference and examples to client libraries.

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 54 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2080)

File: src/writeData/clients/R/description.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+For more detailed and up to date information check out the <a href="https://github.com/influxdata/influxdb-client-r" target="_blank" rel="noreferrer">GitHub Repository</a>
+
+##### Install Package
+
+Released version published on CRAN
+
+```
```
Comment (russorat) [inline src/writeData/clients/R/description.md:7]:
```suggestion
```R
```

File: src/writeData/clients/R/description.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+install.packages("influxdbclient")
+```
+
+Latest development version
+
+```
```
Comment (russorat) [inline src/writeData/clients/R/description.md:13]:
```suggestion
```R
```

File: src/writeData/clients/R/write.example [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    humidity = c(47, 59),
+    time = c(Sys.time(),Sys.time())
+)
+
+client$write(data, bucket = "<%= bucket %>", precision = "ms",
+             measurementCol = "name",
+             tagCols = c("sensor_id"),
+             fieldCols = c("temperature", "humidity"),
+             timeCol = "time")
+
```
Comment (hoorayimhelping) [inline src/writeData/clients/R/write.example:14]: i don't know `R` but is this spacing correct? looks like the bottom block has extra tabs

**Review by russorat — APPROVED** [review]
LGTM

**Review by hoorayimhelping — APPROVED** [review]

**Pull request merged**